packages/zoe/src/zoeService/utils.d.ts

```diff
@@ -104,13 +104,16 @@ export type StartInstance = <SF>(
   issuerKeywordRecord?: Record<Keyword, Issuer<any>>,
   // 'brands' and 'issuers' need not be passed in; Zoe provides them as StandardTerms
   terms?: Omit<StartParams<SF>['terms'], 'brands' | 'issuers'>,
+  // @ts-expect-error XXX
```

Comment:
hm. The other `@ts-expect-error` in this PR seem to have local impact, but this one is imported in various places.

Seems like I should look into this more closely, but I'm not sure it's the best use of time.

in case anyone else finds a moment to take a look at this PR: cc @mhofman @gibson042

Reply:
to be clear this is maintaining earlier behavior, since the .d.ts isn't checked. It just makes it so that when someone does `skipLibCheck: false` this won't register as an error. It also gives a clue to someone working in this file that there's a problem here